state-transition/core/state_processor_genesis.go

```diff
@@ -125,7 +120,6 @@ func (sp *StateProcessor[_]) InitializePreminedBeaconStateFromEth1(
 
 	// Setup a bunch of 0s to prime the DB.
 	for i := range sp.cs.HistoricalRootsLimit() {
-		//#nosec:G701 // won't overflow in practice.
```

Comment:
@calbera. question: why we don't have this anymore?

Reply:
Not needed, outdated nosec